README.md

```diff
@@ -64,7 +64,13 @@ Before diving into LearnHub.mk, make sure you have Node.js installed on your mac
 
        yarn dev
 
-4. Open [localhost:3000](http://localhost:3000) in your browser to explore the app.
+4. Install specific version of Node.js
+
+           nvm install $(cat .nvmrc)
```

Comment:
This should be the the first thing done in the project, not the last.

Reply:
Additionally, please include instructions for Windows Linux and Mac

Reply:
Other things what we need to install with Node are already included before

Reply:
You should add the Node version in a 
## Prerequisites

- Node.js version:


and also add they should run `nvm use` before npm install

Reply:
I make some changes in Readme.md file and like i research we dont need explicitly run `nvm use` after installing the specific Node.js version using `nvm install $(cat .nvmrc)`. The `nvm use` command is automatically handled by the `.nvmrc` file when you navigate to the project.